src/libraries/System.Threading.ThreadPool/src/ApiCompatBaseline.txt

```diff
@@ -0,0 +1,11 @@
+Compat issues with assembly System.Threading.ThreadPool:
+CannotRemoveAttribute : Attribute 'System.Runtime.Versioning.UnsupportedOSPlatformAttribute' exists on 'System.Threading.RegisteredWaitHandle' in the contract but not the implementation.
```

Comment:
No, it's the other way around. 

* CoreLib does not have the attribute (it checks the `Thread.IsThreadStartSupported` flag at runtime).  

* The non-experimental ref assembly always has the attribute.  (So there is always a CannotRemoveAttribute issued)

* The experimental ref  assembly does not have the attribute. (So here the implementation matches the contract)

Reply:
> The non-experimental ref assembly always has the attribute. (So there is always a CannotRemoveAttribute issued)

Why wasn't that baseline required before then? Because the implementation had it before at build time and now it doesn't anymore?

Reply:
> > The non-experimental ref assembly always has the attribute. (So there is always a CannotRemoveAttribute issued)
> 
> Why wasn't that baseline required before then? Because the implementation had it before at build time and now it doesn't anymore?

yes. previously both the implementation and the ref assembly had the attribute